### PR TITLE
Bump requirement on Pillow to 9.2.0

### PR DESCRIPTION
**Pull request by rocky**
Testing shows that we need this to read sunflowers.jpg and possibly do ColorNegate of that.

### CODE REVIEW

**Review by TiagoCavalcante — APPROVED** [review]
LGTM

File: setup.py [hunk 1]
```diff
@@ -43,7 +43,8 @@
     "Mathics-Scanner >= 1.3.0.dev0",
     # Pillow 9.1.0 supports BigTIFF with big-endian byte order.
     # ExampleData image hedy.tif is in this format.
-    "pillow >= 9.1.0",
+    # Pillow 9.2 handles sunflowers.jpg
+    "pillow >= 9.2.0",
```
Comment (mmatera) [inline setup.py:47]: I am finding problems in installing pillow 9.4.0  on anaconda / pyston 2.3.4 (3.8.12)

**Comment by rocky** [comment]
And 9.2.0? or 9.3.0?

**Comment by mmatera** [comment]
The problems that I found were related to setuptools. After upgrading it, everything worked.

**Pull request merged**